docs/source/changelog.rst

```diff
@@ -11,6 +11,7 @@ Array
 
 DataFrame
 +++++++++
+- Bugfix to allow column assignment of pandas datetimes(:pr:`3164`)`Max Epstein`_
```

Comment:
Maybe a space between the `)` and the backtick. I think Sphinx might complain otherwise